### ISSUE TITLE
Use /tmp directory for base64 images

### ISSUE THREAD

**Issue by ahallock** [vkononov/fillable-pdf#31]
I was trying to use this on Lambda and it's read-only. Using /tmp would be more friendly for this type of env. Can submit a PR if you want

**Comment by vkononov** [comment]
@ahallock I am not sure what you're talking about. The gem allows you to specify both input and output directories yourself. Can you elaborate on what you're trying to do and what is not working the way you need?

**Comment by ahallock** [comment]
@vkononov I mean when setting a base64 image field:
``` ruby
  def set_image_base64(key, base64_image_data)
    tmp_file = SecureRandom.uuid
    File.binwrite(tmp_file, Base64.decode64(base64_image_data))
    set_image(key, tmp_file)
 end
```

Would like to use the tmp directory for this file, like:

`tmp_file = "/tmp/#{SecureRandom.uuid}"`

**Comment by vkononov** [comment]
Now it makes sense. Thank you for clarifying. I may need to check to see how that would work on Windows, but that's a great suggestion. I'll implement that in a bit. Thanks again, @ahallock.

**Comment by ahallock** [comment]
@vkononov yeah sorry, it was late and didn't word that too well. Thanks for a great gem -- it's very easy to use. Good call on using an OS-independent solution -- maybe `Dir.tmpdir()`
